### DISCUSSION TITLE
Where is showImagePicker method?

### DISCUSSION THREAD

**Discussion by ngotrongphuc** [react-native-image-picker/react-native-image-picker#1748] [Q&A] [answered]
Why is there only 2 methods _launchCamera_ and _launchImageLibrary_?

**Comment by Johan-dutoit** [comment, marked as answer]
To make this library easier to maintain, the user is now expected to create their own version of `showImagePicker` and reduce the footprint of this library.

You can use [react-native-action-sheet](https://github.com/expo/react-native-action-sheet) to create your own picker.
